homeassistant/components/vconnex/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""The Vconnex integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import (
+    DOMAIN,
+    PLATFORMS,
+)
+
+from .vconnex_wrap import init_sdk, release_sdk
+
+LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Async setup hass config entry."""
+    if DOMAIN not in hass.data:
+        hass.data.setdefault(DOMAIN, {})
```

Comment:
```suggestion
    hass.data.setdefault(DOMAIN, {})
```
setdefault already checks for the key.